index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Bootstrap CDN -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ"
+      crossorigin="anonymous"
+    />
+    <!-- CSS Connection -->
+    <link rel="stylesheet" href="./index.css">
+    <title>Awesome Books</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <header class="">
+      <nav class="px-5 navbar navbar-expand-lg bg-body-tertiary">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">Awesome Books</a>
+          <ul class="navbar-nav w-25 flex-row justify-content-between">
+            <li class="nav-item">
+              <a
+                class="nav-link active text-primary"
+                id="list"
+                aria-current="page"
+                href="#"
+                >List</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active" id="add-new" href="#">Add new</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active" id="contact" href="#">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!-- Date -->
+    <time
+      class="fw-bold text-right w-100 d-flex justify-content-end pe-5 pt-2"
+    ></time>
+    <!-- Main -->
+    <main class="py-5"></main>
+
+    <!-- Footer -->
+    <footer
+      class="bg-light position-fixed fixed-bottom d-flex justify-content-around align-item-center"
+    >
+      <p>&copy;Copyright</p>
+    </footer>
+    <!-- Bootstrap CDN -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe"
+      crossorigin="anonymous"
+    ></script>
+    <!-- CSS Connection -->
+
+    <!-- <script src="js/date.js"></script> -->
+    <!-- <script src="js/script-class.js"></script> -->
+    <!-- <script src="js/spa.js"></script> -->
```

Comment:
- [ ] I think it would be better to remove the commented lines in your codes to make it more cleaner and professional 👍 👍